app/utils/index.ts

```diff
@@ -22,7 +22,7 @@ class Generator {
 
   constructor(config?: IDGeneratorConfig) {
     const { start, end, step } = this.getConfig(config);
-    this.newGen = this.idGenerator(start, end, step);
+    this.newGen = this.generate(start, end, step);
```

Comment:
```suggestion
    this.idGenerator = this.generate(start, end, step);
```

Not a big deal; but how about?

Reply:
Done!